src/client/auth-service.js

```diff
@@ -15,7 +15,8 @@ export function login(nextUrl) {
       redirectUrl: window.AUTH0_LOGIN_CALLBACK,
       responseType: 'code',
       params: {
-        state: nextUrl
+        state: nextUrl || '/',
```

Comment:
ok this is what fixed it - state being updated with the nextUrl was returning a weird letter string in some cases